.github/workflows/reusable_checks.yml

```diff
@@ -52,6 +52,14 @@ jobs:
     - name: Check Python formatting
       run: cmake --build build --target black-format-check
 
+    - name: Run check-license
+      run: |
+        bash ./check_license/check-headers.sh ./ "Apache-2.0 WITH LLVM-exception" -v
```

Comment:
can't we use the new custom make targets `copyright-format` and `check-license`?

Reply:
bump - what about this one?

Reply:
I kept only the job responsible for activating check-headers script